src/main/java/org/folio/dew/batch/bursarfeesfines/service/BursarFilterEvaluator.java

```diff
@@ -16,6 +18,7 @@
 import org.folio.dew.domain.dto.BursarExportFilterPatronGroup;
 import org.folio.dew.domain.dto.BursarExportFilterServicePoint;
 import org.folio.dew.domain.dto.bursarfeesfines.AccountWithAncillaryData;
+import org.springframework.cglib.core.Local;
```

Comment:
What's this for?

Reply:
Typo, my bad. I just pushed a commit to fix it